crates/wasmcloud-host/src/dispatch.rs

```diff
@@ -362,17 +368,11 @@ fn invocation_from_callback(
     provider_id: &str,
     payload: &[u8],
 ) -> Invocation {
-    let link_name = if bd.trim().is_empty() {
-        // Some actor SDKs may not specify a link field by default
-        "default".to_string()
-    } else {
-        bd.to_string()
-    };
     let target = if ns.len() == 56 && ns.starts_with("M") {
         WasmCloudEntity::Actor(ns.to_string())
     } else {
         WasmCloudEntity::Capability {
-            link_name,
+            link_name: bd.to_string(),
```

Comment:
Shouldn't this be `link_name` rather than the unmodified `bd` ?

Reply:
That change was because the logic that created the `link_name` variable was moved up to `wapc_host_callback`, there's no `link_name` in that function anymore